src/seedu/addressbook/common/Utils.java

```diff
@@ -12,7 +12,7 @@
     /**
      * Returns true if any of the given items are null.
      */
-    public static boolean isAnyNull(Object... items) {
+    public static boolean isAnyNull(Collection<?> items) {
```

Comment:
Why this change is needed? `Varargs` doesn't follow the coding convention? If you really want to use `Collection<?>`, you should also modify the methods below to keep them aligned.